src/fal/packages/_run_hook.py

```diff
@@ -0,0 +1,54 @@
+from re import L
+from logbook import StreamHandler
+import json
+import sys
+from argparse import ArgumentParser, Namespace
+from pathlib import Path
+from typing import Any, Dict
+
+
+def reconfigure_handlers():
+    from logbook import INFO, StreamHandler
+    from dbt.logger import GLOBAL_LOGGER as logger
+
+    # Reconfigure DBT's log handlers to output to
+    # stdout so we can stream it back to the parent
+    # process.
+    logger.disabled = False
```

Comment:
https://github.com/fal-ai/fal/blob/c70a2f4b37621accbd256d85de5e5181e866cf16/src/fal/cli/cli.py#L33-L38

Reply:
> ATIL: what is new dbt logging?

https://github.com/dbt-labs/dbt-core/blob/v1.0.0/core/dbt/events/README.md#adapter-maintainers

We had not migrated because 0.X dbt did not have the new logging available. I will get it done soon!

Reply:
I think the initial idea of just adding a `TODO` comment in here to remember to take a look is enough. To not block this PR at all.

Reply:
I've unified both fal.cli and this part, so once this is landed and the changes are applied it will fix both of them I think. Thanks for mentioning about this btw!